metadata.rb

```diff
@@ -380,6 +379,14 @@
           :description => "LocationDomainIds for ndb api nodes (namenodes)",
           :type => 'hash'
 
+attribute "ndb/mysql_socket",
+          :description => "Location of the MySQL unix socket"
+          :type => "string"
+
+attribute "ndb/mysql_port",
+          :description => "Port on which the MySQL server binds to"
+          :type => "string"
+
```

Comment:
These are defined in kagent-chef
https://github.com/logicalclocks/kagent-chef/blob/master/attributes/default.rb#L76

They are used by other recipes (like airflow and others) that i don't think depend on ndb-chef.

Reply:
I guess we should move them to conda-chef.

Reply:
Actually I think this is the source of the socket problem. Kagent had a functionality to query the `ndbinfo` table through the socket. I suppose it was there for monitoring purposes. I think that's why the socket was placed in `/tmp` so that Kagent could read it. 
Now with the `mysqld_exporter` the Kagent functionality is not needed anymore. This PR: https://github.com/logicalclocks/kagent-chef/pull/103 removes it from Kagent. 

I also checked all the cookbooks that somehow need to interact with the DB, all of them depend on the ndb cookbook. So I think it's cleaner and more intuitive if the these attributes are in the ndb cookbook